parts/k8s/kubernetesagentcustomdata.yml

```diff
@@ -184,25 +184,25 @@ runcmd:
 - set -x
 - . /opt/azure/containers/provision_source.sh
 - apt_get_update() { for i in $(seq 1 100); do apt-get update 2>&1 | grep -x "[WE]:.*"; [ $? -ne 0  ] && break || sleep 1; done; echo Executed apt-get update $i times; }
-- retrycmd_if_failure 120 1 nc -zuw1 $(grep nameserver /etc/resolv.conf | cut -d \  -f 2) 53
-- retrycmd_if_failure 120 1 nc -zw1 aptdocker.azureedge.net 443
+- retrycmd_if_failure 120 1 5 nc -zuw1 $(grep nameserver /etc/resolv.conf | cut -d \  -f 2) 53
```

Comment:
all prior instances of `retrycmd_if_failure` usage have been adapted to include the new 3rd arg for timeout